Providers/nxOMSAutomationWorker/MSFT_nxOMSAutomationWorkerResource.h

```diff
@@ -29,9 +29,7 @@ typedef struct _MSFT_nxOMSAutomationWorkerResource /* extends OMI_BaseResource *
     MI_Instance __instance;
     /* OMI_BaseResource properties */
     /* MSFT_nxOMSAutomationWorkerResource properties */
-    /*KEY*/ MI_ConstStringField WorkspaceId;
```

Comment:
@lagalbra  Any concern with this ^ ? @D1v38om83r we could still keep the workspaceId as a seperate parameter if needed...